Sources/TZSettingsView/TZSettingsCell/TZSettingsCellContentView.swift

```diff
@@ -27,15 +27,32 @@ struct TZSettingsCellContentView: View {
              The image to be displayed in the setting cell.
              */
             configuration.image
-                .frame(maxWidth: 20, maxHeight: 20)
+                .frame(maxWidth: configuration.theme?.imageWidth, maxHeight: configuration.theme?.imageHeight)
+                .scaleEffect(((configuration.theme?.isAnimated ?? false) ?
+                              configuration.theme?.scaleEffectValue :
+                                1.0) ?? 1.0)
+                .animation(configuration.animation, value: (configuration.theme?.isAnimated ?? false))
```

Comment:
Is `configuration.theme?.isAnimated` State? when the isAnimated value will change?